README.md

```diff
@@ -242,21 +242,22 @@ The common typeface classifications are:
 * FontFont (FF)
 * Google Design
 * Indian Type Foundry (ITF)
+* Klim Type Foundry
```

Comment:
This is a good addition. I like her work. 

I was thinking we could add a seperate section of noteworthy independent type foundries. Please raise a separate pull request with that.

Reply:
Added to the list. 
Thank you.